synapse/res/templates/_base.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{% block title %}{% endblock %}</title>
+    <link rel="stylesheet" href="./style.css">
+    {% block header %}{%  endblock %}
+</head>
+<body>
+<header class="mx_Header">
+    {% if app_name == "Riot" %}
+        <img src="http://riot.im/img/external/riot-logo-email.png" width="83" height="83" alt="[Riot]"/>
+    {% elif app_name == "Vector" %}
+        <img src="http://matrix.org/img/vector-logo-email.png" width="64" height="83" alt="[Vector]"/>
+    {% elif app_name == "Element" %}
+        <img src="https://static.element.io/images/email-logo.png" width="83" height="83" alt="[Element]"/>
+    {% else %}
+        <img src="http://matrix.org/img/matrix-120x51.png" width="120" height="51" alt="[matrix]"/>
```

Comment:
I just realized that `app_name` is generally only available for "email" type templates, not all templates. So this will default to the Matrix version *most* of the time. I don't know if this is a problem or not.

Reply:
How does jinja2 behave if the `app_name` variable isn't provided---treat it as `None` when evaluating the template?

Reply:
@gsouquet and I discussed this and we think it is OK (and @nadonomy agreed it was fine).

In the future we should probably make the [`email.app_name`](https://matrix-org.github.io/synapse/develop/usage/configuration/config_documentation.html#email) config apply more globally.